tests/optional_kernel_features/kernel_features_common.h

```diff
@@ -72,11 +72,10 @@ inline void use_feature_function_non_decorated() {
 template <typename T>
 inline void use_feature_function_non_decorated_with_accessor(
     const sycl::accessor<bool, 1> &acc) {
-  unsigned long long temp = 42;
+  unsigned long long temp = acc[0];
   T feature1(temp);
-  T feature2(temp);
-  feature1 += 42;
-  acc[0] = (feature1 == feature2);
+  feature1 += 1;
```

Comment:
As we have only one variable named `feature`, I suggest dropping the counter at the end of the name.